AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml

```diff
@@ -69,9 +73,15 @@
                 <!--/.nav-collapse -->
             </div>
         </nav>
+        <div class="row logo-background" @(descriptor.ActionName == "Index" && controllerName == "Home" ?  "" : " style=display:none;" )>
+            <div class="col-md-12">
+                <div class="center"><img src="~/images/Web-Logo.png" alt="Logo image for the project" class="img-responsive center-block" /></div>
+                <h3 class="text-center sub-title">Get connected. Work together. Be prepared.</h3>
+            </div>
+        </div>

```

Comment:
This hero div really should live in `Home/Index.cshtml`. It's OK if that means the layout changes a little. It's more important for the code to be logically clean.